lodash.src.js

```diff
@@ -9591,6 +9591,30 @@
     }
 
     /**
+     * Converts `string` to pascal case.
+     * See [Wikipedia](https://en.wikipedia.org/wiki/CamelCase) for more details.
+     *
+     * @static
+     * @memberOf _
+     * @category String
+     * @param {string} [string=''] The string to convert.
+     * @returns {string} Returns the pascal cased string.
+     * @example
+     *
+     * _.pascalCase('Foo Bar');
+     * // => 'FooBar'
+     *
+     * _.pascalCase('--foo-bar');
+     * // => 'FooBar'
+     *
+     * _.pascalCase('__foo_bar__');
+     * // => 'FooBar'
+     */
+    function pascalCase(string) {
+      return capitalize(camelCase(string));
```

Comment:
With the latest lodash you want to use `upperFirst` here instead of `capitalize`.